gemato/openpgp.py

```diff
@@ -183,11 +184,12 @@ class OpenPGPEnvironment(OpenPGPSystemEnvironment):
     or use as a context manager (via 'with').
     """
 
-    __slots__ = ['_home']
+    __slots__ = ['_home', '_env']
 
     def __init__(self, debug=False):
         super(OpenPGPEnvironment, self).__init__(debug=debug)
         self._home = tempfile.mkdtemp()
+        self._env = {}
```

Comment:
I suppose you want to pull `ENVIRON_WHITELIST` from `os.environ` here.

Reply:
Not necessary, but default policy could be indeed to add them. Still, that does not solve the Portage values provided in it's configuration file.